src/ucs/stats/stats.h

```diff
@@ -19,10 +19,61 @@ BEGIN_C_DECLS
 
 /** @file stats.h */
 
+/* The maximum size of a class name */
+#define MAX_CLS_NAME_SIZE 50
+
+/* The maximum number of UCX aggregate-sum counters */
+#define MAX_NUM_OF_STATS_AGGREGATE_SUM_COUNTERS 32
+
+/* The maximum number of statistics classes */
+#define MAX_NUM_OF_STATS_CLASSES 32
+
+typedef struct tag_ucs_stats_counter {
+    /* Counter class name */
+    const char *class_name;
+
+    /* Counter name */
+    const char *name;
+
+    /* Counter value */
+    uint64_t value;
+
+} ucs_stats_counter_s;
```

Comment:
ucs_stats_counter_t

Reply:
This type name is already occupied in UCX:
$ git grep ucs_stats_counter_t | grep typedef
src/ucs/stats/stats_fwd.h:typedef uint64_t                          ucs_stats_counter_t;        /* Stats counter*/

We agreed on ucs_stats_aggrgt_sum_counter_name_t I think.